src/stack/Interface.java

```diff
@@ -0,0 +1,13 @@
+package stack;
+
+public interface Interface<T> {
```

Comment:
Please change the name of interface to Stack (Interface is not a proper name)

Reply:
The changes have been made